src/CarouselCardsRow.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { createGlobalStyle } from 'styled-components'
+
+import cardPropTypes from './modelPropTypes'
+import Card from './Card'
+import Slide from 'react-slick'
+
+const CarouselCardsRow = (props) => {
+  const { CardClass, cards, slideSettings } = props
+  const { className, cardContainerClassName, speciesIconHeight, imageIconHeight, hoverColour } = props
+  slideSettings.slidesToShow = slideSettings.slidesToShow >= cards.length ? cards.length : slideSettings.slidesToShow
+
+  const CardContainer = styled.div`
+    /* border-radius: 8px; */
```

Comment:
Remove comment. If we don’t need it, we don’t need it.

Reply:
Yes. I also deleted the original comment in `ResponsiveCardsRow`.